plugins/org.eclipse.elk.alg.vertiflex/src/org/eclipse/elk/alg/vertiflex/p1yplacement/NodeYPlacer.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 
+ *******************************************************************************/
+package org.eclipse.elk.alg.vertiflex.p1yplacement;
+
+import org.eclipse.elk.alg.vertiflex.InternalProperties;
+import org.eclipse.elk.alg.vertiflex.VertiFlexLayoutPhases;
+import org.eclipse.elk.alg.vertiflex.options.VertiFlexOptions;
+import org.eclipse.elk.core.alg.ILayoutPhase;
+import org.eclipse.elk.core.alg.LayoutProcessorConfiguration;
+import org.eclipse.elk.core.options.CoreOptions;
+import org.eclipse.elk.core.util.IElkProgressMonitor;
+import org.eclipse.elk.graph.ElkNode;
+
+/**
+ * Node placer to position nodes vertically. Nodes that have a vertical constraint are placed according to that 
+ * constraint and other nodes are positioned automatically according to their position in the tree.
+ *
+ */
+public class NodeYPlacer implements ILayoutPhase<VertiFlexLayoutPhases, ElkNode> {
+
+    private double layerDistance;
+    private double nodeNodeSpacing;
+    private IElkProgressMonitor myProgressMonitor;
```

Comment:
It seems that this variable is never really used. It is only used in the constructor.